src/entities/raw-data-providers/jormungandr-api.js

```diff
@@ -148,17 +151,19 @@ class JormungandrApi implements RawDataProvider {
     // TODO: wait for jormungandr to be updated - mocking out some info for now instead
     const resp = await this.getJson('/node/stats')
     const { data } = resp
+    const epochNumber = Math.floor(data.lastBlockHeight / this.networkSlotsPerEpoch);
+    const slotNumber = data.lastBlockHeight % this.networkSlotsPerEpoch;
```

Comment:
I'm not sure this is even right. When I wrote the original code I didn't realize that we could have empty slots/etc so this might give bad numbers, which may cause it to crash or behave weird if it fully syncs to a chain that had empty slots.